app/index.js

```diff
@@ -2,14 +2,16 @@
 require('dotenv').config();
 
 import config from './config';
-import Server from './services/server';
+import initServer from './services/server';
 
 var options = {
   connection: config.connection
 };
 
 // Start API server
-var server = new Server(options);
-server.start(() => {
-  // Started.
+initServer(options, (err, server) => {
```

Comment:
I fixed plugins.js, but I still had trouble with having the plugins register before server start when moving to synchronous initialization.

```
/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/hoek/lib/index.js:736
    throw new Error(msgs.join(' ') || 'Unknown error');
    ^

Error: Cannot add onPreStart (after) extension after the server was initialized
    at Object.exports.assert (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/hoek/lib/index.js:736:11)
    at module.exports.internals.Plugin.internals.Plugin._ext (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/hapi/lib/plugin.js:481:10)
    at module.exports.internals.Plugin.internals.Plugin.ext (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/hapi/lib/plugin.js:453:14)
    at Object.exports.register (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/good/lib/index.js:34:12)
    at Object.target [as register] (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/joi/lib/object.js:77:34)
    at each (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/hapi/lib/plugin.js:318:14)
    at iterate (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/items/lib/index.js:36:13)
    at done (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/items/lib/index.js:28:25)
    at Object.exports.register (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/app/plugins/rra-osm-p2p-server.js:64:3)
    at Object.target [as register] (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/joi/lib/object.js:77:34)
    at each (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/hapi/lib/plugin.js:318:14)
    at iterate (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/items/lib/index.js:36:13)
    at Object.exports.serial (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/items/lib/index.js:39:9)
    at module.exports.internals.Server.internals.Plugin.register (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/hapi/lib/plugin.js:321:11)
    at /Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/app/services/plugins.js:32:16
    at process.nextTick (/Users/marc/code/WorldBank-Transport/Rural-Road-Accessibility/node_modules/hoek/lib/index.js:854:22)
```

What do you suggest?